docs/wired-client-architecture.md

```
@@ -0,0 +1,367 @@
+**Symptom:**
+Same card sent multiple times
+
+**Solution:**
+- Controlled by `CARD_SEND_COOLDOWN_MS`
+- Wired Client: 500ms
+- WiFi Client: 10000ms
+
+### HTTP POST Failures
+
+**Symptom:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)